examples/k8s-example.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: node-labels-to-files
+  namespace: default
+
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: node-labels-to-files
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  verbs:
+  - get
```

Comment:
needs `list` and `watch` for the `always` mode, if you are feeling like it you can open a PR or I'll do that if I do not see a PR by EoD my time.